app/workers/notify_vacancy_on_slack_worker.rb

```diff
@@ -0,0 +1,16 @@
+require "slack_notifier"
+
+class NotifyVacancyOnSlackWorker
+  include Sidekiq::Worker
+  include Rails.application.routes.url_helpers
+
+  def perform(vacancy_id)
+    vacancy = Vacancy.find(vacancy_id)
+    message = %Q(
```

Comment:
I was thinking, maybe the message shouldn't be inside the the slack notifier? Usually workers should just delegate to another object. He shouldnt "think"